conversationgenome/utils/config.py

```diff
@@ -151,14 +151,14 @@ def add_miner_args(cls, parser):
         "--blacklist.force_validator_permit",
         action="store_true",
         help="If set, we will force incoming requests to have a permit.",
-        default=False,
+        default=True,
     )
 
     parser.add_argument(
         "--blacklist.allow_non_registered",
         action="store_true",
         help="If set, miners will accept queries from non registered entities. (Dangerous!)",
-        default=False,
+        default=True,
```

Comment:
I believe this should be set to false for security. Can you explain why we would want this to default to True?

Reply:
I'm going to change this one to False and merge your PR. If you think it should be true, we can discuss it and do another PR.

Reply:
oh, was my mistake, thought it was `blacklist_non_registered`